app/views/users/new.html.erb

```diff
@@ -3,47 +3,47 @@
 
   <div class="signup-breadcrumb">
      <div style="background-color:#eee;">
-         Create Account
+         Create Account/Créer le Compte 
      </div>
      <div>
-         Additional Information
+         Additional Information/Information Additionnelle
      </div>
   </div>
-  <div class="signup-title">Create your personal account</div>
+  <div class="signup-title">Create your personal account/Créer le Compte Personel</div>
 
   <%= form_for @new_user do |f| %>
 
-    <%= f.label :username %>
+    <%= f.label :username , "Username/Nom d'Utilisateur"%>
     <%= content_tag :span, @new_user.errors[:username][0], class: 'form-error' %>
     <%= content_tag :span, @new_user.errors[:username][1], class: 'form-error',
         style: 'display: block;' if @new_user.errors[:username][1] %>
     <%= f.text_field :username, autofocus: true %>
 
-    <%= f.label :email %>
+    <%= f.label :email , "Email/Adresse Courriel" %>
     <%= content_tag :span, @new_user.errors[:email].first, class: 'form-error' %>
     <%= f.email_field :email %>
 
-    <%= f.label :password %>
+    <%= f.label :password ,"Password/Mot de passo" %>
     <%= content_tag :span, @new_user.errors[:password][0], class: 'form-error' %>
     <%= content_tag :span, @new_user.errors[:password][1], class: 'form-error',
         style: 'display: block;' if @new_user.errors[:password][1] %>
     <u>Use at least one uppercase letter, one lowercase letter,
-      one number and be eight characters long.</u>
+      one number and be eight characters long./ Utilisez au moins une lettre majuscule, une lettre minuscule, un chiffre et il doit être huit caractères de long.</u>
     <%= f.password_field :password %>
 
-    <%= f.label :password_confirmation, "Password Confirmation" %>
+    <%= f.label :password_confirmation, "Password Confirmation/ Confirmation de Mot de Passe" %>
     <%= content_tag :span, @new_user.errors[:password_confirmation][0], class: 'form-error' %>
     <%= content_tag :span, @new_user.errors[:password][1], class: 'form-error',
         style: 'display: block;' if @new_user.errors[:password][1] %>
     <%= f.password_field :password_confirmation %>
 
     <div id="terms_and_conditions_box" >
       <%= f.check_box :terms_and_conditions, :style => "float:right;" %>
-      <%= f.label :terms_and_conditions, :class => "checkbox inline" do "I agree to the #{link_to "terms and condition", "terms_and_conditions.pdf", target: "blank"}".html_safe end %>
+      <%= f.label :terms_and_conditions, :class => "checkbox inline" do "I agree to the #{link_to "terms and condition", "terms_and_conditions.pdf", target: "blank"}", "J\'accepte les #{link_to "termes et conditions", "terms_and_conditions.pdf", target: "blank"}".html_safe end %>
```

Comment:
This doesn't quite work the way you want, you need to have it all in the same string, no "," between